src/data/Fees.js

```diff
@@ -1,7 +1,7 @@
 export const SECONDS_PER_WEEK = 60 * 60 * 24 * 7;
 
 const FEES = {
-  421611: [],
+  421613: [],
   42161: [
```

Comment:
Could update these to network names [ARBITRUM_GOERLI] and [ARBITRUM]

Reply:
I tried that, but it throws a weird error when I do.  Something about the order in which the variables are initialized